cli/tests/program.rs

```diff
@@ -577,6 +591,8 @@ fn test_cli_program_deploy_with_authority() {
         program_pubkey,
         upgrade_authority_index: Some(1),
         new_upgrade_authority: None,
+        sign_only: false,
+        dump_transaction_message: false,
```

Comment:
This probably needs the `blockhash_query` too
```suggestion
        sign_only: false,
        dump_transaction_message: false,
        blockhash_query: BlockhashQuery::default(),
```